examples/react/filters/src/main.tsx

```diff
@@ -42,6 +43,8 @@ const _features = tableFeatures({
   rowPaginationFeature,
 })
 
+type TestType = ExtractFeatureTypes<typeof _features, 'Column'>
```

Comment:
This is currently unknown. It seems like more needs to be done to override the interface of the Columns maybe, is this correct?